.github/workflows/pull_request.yaml

```
@@ -0,0 +1,41 @@
+name: Build Chain
+
+on: [pull_request]
+
+jobs:
+  build-chain:
+    concurrency:
+      group: pull_request-${{ github.head_ref }}
+      cancel-in-progress: true
+    strategy:
+      matrix:
+        java-version: [8, 11]
+      fail-fast: true
+    runs-on: ubuntu-latest
+    name: Maven Build
+    steps:
+      - name: Set up JDK
+        uses: actions/setup-java@v1
+        with:
+          java-version: ${{ matrix.java-version }}
+      # See https://docs.github.com/en/actions/guides/building-and-testing-java-with-maven#caching-dependencies
+      - name: Cache Maven packages
+        uses: actions/cache@v2
+        with:
+          path: ~/.m2
+          key: ${{ runner.os }}-${{ matrix.java-version }}-m2-${{ hashFiles('**/pom.xml') }}
+          restore-keys: ${{ runner.os }}-${{ matrix.java-version }}-m2
+      - name: Build Chain ${{ matrix.java-version }}
+        id: build-chain
+        uses: kiegroup/github-action-build-chain@v2.6.3
```

```suggestion
        uses: kiegroup/github-action-build-chain@v2.6.4
```
use latest version of the tool since it solves problems with mapping

---

DONE